tests/Unit/PrestaShopBundle/Utils/FloatParserTest.php

```diff
@@ -97,6 +97,29 @@ public function provideValidStrings()
             ['', 0.0],
             ['   ', 0.0],
         ];
+
+        // cldr minus signs
+        $cldr_minus_signs = $this->str_split_unicode('-‒⁻₋−➖﹣－');
```

Comment:
Could you please only fill the array above and don't create something dynamic.
Seems you copy & paste a method. We are not using snake case anywhere :thinking:

Reply:
I was thinking it is easier to update the test case when that string can be copied from cldr definition if it changes there in the future.

continued the `str_split` to have `_unicode` is where the naming convention comes from.. mb_str_split would work also but it is only available from php7.4 forward...

I'll modify the test case in a bit.

Reply:
done in d56f3e8

Reply:
Tests have to be as simple as possible, more when it concerns data providers. (That's not something you doing when using TDD)
Filling the array will be enough.